src/components/ContentHighlights/ZeroState/ZeroStateHighlights.jsx

```diff
@@ -13,11 +15,34 @@ import ContentHighlightStepper from '../HighlightStepper/ContentHighlightStepper
 import { ContentHighlightsContext } from '../ContentHighlightsContext';
 import { useContentHighlightsContext } from '../data/hooks';
 import { BUTTON_TEXT } from '../data/constants';
+import { EnterpriseAppContext } from '../../EnterpriseApp/EnterpriseAppContextProvider';
+import EVENT_NAMES from '../../../eventTracking';
 
-const ZeroStateHighlights = ({ cardClassName }) => {
+const ZeroStateHighlights = ({ enterpriseId, cardClassName }) => {
   const { openStepperModal } = useContentHighlightsContext();
+  const {
+    enterpriseCuration: {
+      enterpriseCuration: {
+        highlightSets,
+      },
+    },
+  } = useContext(EnterpriseAppContext);
   const isStepperModalOpen = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal.isOpen);
-
+  const handleNewHighlightClick = () => {
+    openStepperModal();
+    const trackInfo = {
+      highlight_sets: highlightSets,
```

Comment:
if this event is firing from the "zero-state" component, `highlight_sets` is always going to be an empty array here, so I think it can be omitted along with `number_of_highlight_sets`.

Reply:
Put this in with the understanding that if the highlight sets do exist but they are firing from zero state, it gives us the ability to view it in the logs to troubleshoot future potential errors that may occur